src/components/FullscreenGrid/ContentShell.tsx

```diff
@@ -34,11 +35,13 @@ export const ContentShellBase = ({
   dispatch,
   mainSidePanel,
   mainSidePanelClass,
+  mainSidePanelIsBorderless = false,
   mainSidePanelIsExpanded,
 }: Props) => {
   return (
     <React.Fragment>
       <SidePanel
+        borderless={mainSidePanelIsBorderless}
```

Comment:
This needs a test to make sure `SidePanel` gets configured with the value of the prop `mainSidePanelIsBorderless`.